apps/server/src/modules/oauth/controller/oauth.controller.ts

```diff
@@ -0,0 +1,23 @@
+import { Controller, Get, Param, Query, Res } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
+import { OauthUc } from '../uc/oauth.uc';
+import { AuthorizationQuery } from './dto/authorization.query';
+import { OAuthResponse } from './dto/oauth.response';
+
+@ApiTags('Oauth')
+@Controller('oauth')
+export class OauthController {
+	constructor(private readonly oauthUc: OauthUc) {}
+
+	@Get(':systemid')
```

Comment:
this is code endpoint, give it that name
```suggestion
	@Get('code/:systemid')
```

Reply:
this is oauth start endpoint, and it happens to be the only endpoint and it starts with the code. It does make sense to call it with "code/....." but the next request to the token endpoint should have a redirect_uri of "code/..../token" and that makes no sense. That's why we decided to go with the normal flow, the first step (endpoint) is with no specified name.

Reply:
closing after our conversation in person